packages/core/storage/src/storage.ts

```diff
@@ -17,11 +17,17 @@ alwatrRegisteredList.push({
 /**
  * Elegant powerful micro in-memory document Database with disk backed.
```

Comment:
Elegant micro in-memory json-like storage with disk backed, Faster NoSQL Database written in tiny TypeScript ES module.